### ISSUE TITLE
Merge PDO_*_Ext definitions into PDO class

### ISSUE THREAD

**Issue by ondrejmirtes** [phpstan/php-8-stubs#23]
In php-src there are some special stubs that are not supposed to be used on their own because they don't represent any real class, like: https://github.com/php/php-src/blob/master/ext/pdo_pgsql/pgsql_driver.stub.php

/cc @kocsismate Are you aware of any other special stubs that need special handling in php-src? Thanks!

**Comment by kocsismate** [comment]
Yes, there is `PDO_SQLite_Ext`. But I'm not aware of any other special stub.

**Comment by kocsismate** [comment]
I mean, there is a few weirdness which you may already be aware of:
- SAPIs alias the same function many times (e.g. `getallheaders()`)
- there are a few test extensions: zend_test, dl_test and skeleton
